Gemfile

```diff
@@ -79,13 +79,15 @@ group :development, :test do
   gem 'pry-byebug'
   gem 'rspec-rails', '~> 3.5.2'
   gem 'slim_lint'
+  gem 'teaspoon-mocha'
```

Comment:
Teaspoon places it there in their installation instructions, I believe that enables visiting the `/teaspoon/default` route and running specs in the browser.